### DISCUSSION TITLE
color of cables for npm sensor connection

### DISCUSSION THREAD

**Discussion by spidermas59** [huizebruin/s0tool#137] [General] [answered]
hello,

in your plan you show only two cables with the NPM sensor. on the one I bought, there are 3 cables. I connected it following your doc and not using the black cable that comes out of the npm sensor but the pulse does not report any information.

can you confirm that you should only use 2 cables on the NPM sensor?

![image](https://github.com/huizebruin/s0tool/assets/132600744/836d7c38-0286-405e-8c0a-982c14a4f182)

![image](https://github.com/huizebruin/s0tool/assets/132600744/0d4252c3-deb5-4a93-8036-9466d2ab1b6d)

**Comment by huizebruin** [comment, marked as answer]
https://github.com/huizebruin/s0tool/blob/main/static/assets/npn-watermeter-wemosd1.png

Like this?
It's standing on the readme file on the S0tool Repro on GitHub.
I will post it also on the flash page.
